theano/gpuarray/dnn.py

```diff
@@ -2501,7 +2501,7 @@ def make_node(self, desc, x, hx, y, reserve, w):
 
 class RNNBlock(object):
     """
-    An object that allow us to use CuDNN v5 RNN implementation.
+    An object that allow us to use CuDNN RNN implementation.
     TODO: make an example how to use. You can check Theano tests
```

Comment:
Not in the docstring.  It would be about 100 lines long.  I still think that the tests are a sort of good reference.